src/store/board-service/reducer.ts

```diff
@@ -0,0 +1,33 @@
+import { IBoardReducer } from '../../utils/types/reducers/board-reducer.interface';
+import { LeaderBoardActions } from './actions';
+
+const inititalState: IBoardReducer = {
+	user: [
+		{
+			img: '',
+			score: 0,
+			name: '',
+		},
+	],
+	isModalOpen: true,
+};
+
+export const boardReducer = (state: IBoardReducer, action: any): IBoardReducer => {
```

Comment:
fix an any

Reply:
https://github.com/FiveFastTreck/FT-web/blob/main/src/store/order-service/types.ts - here is an example

Reply:
example is undefined